src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -416,12 +436,12 @@ public NetworkPeer Connect(IPEndPoint endpoint)
                 OneTry = true
             });
 
-            NetworkPeer node = this.networkPeerFactory.CreateConnectedNetworkPeer(this.Network, endpoint, cloneParameters);
+            NetworkPeer peer = this.networkPeerFactory.CreateConnectedNetworkPeer(this.Network, endpoint, cloneParameters);
```

Comment:
I think there should already be a `ConnectionManagerBehavior`  in `this.Parameters` so we need to check this logic.